HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -365,6 +366,7 @@
 
     fragment.MC_JME,
     fragment.MC_BTV,
+    fragment.MC_TRK,
```

Comment:
Both `MC_JME` and `MC_BTV` already include `HLTTrackingSequence`.

Reply:
I wanted to follow the pattern that exists in Run3 (and probably was in Run2), to have a tracking-only path, similar to `MC_ReducedIterativeTracking_v*`

Reply:
Ciao @slava77 my only "doubt" is related to having different MC paths in the "central" Menu that, in the end, will do/trigger "full" reco (or at least full tracking) on all events, no matter what.
Having one additional MC path that does only tracking, like the one you are proposing, in the regular 75e33 Menu will be, in this case, cost free, but I don't see how this will be useful/be used on a real life example?

And, in fact, you have added a dedicated "menu" to run that path only.

Reply:
> > aren't the timing tools allowing to measure the time of a path (non-exclusive) from the full menu (HLT_75e33_cff as in this case)? If so, I still think it's useful to keep this in the main menu.
> 
> I am not aware of that. Do you have more details about it?

I had in mind the regular TimerService `path time_real`; I usually see these in DQM files of the timing reports. (e.g. some [random run3 timing test](https://timing-gui-tsg-steam.app.cern.ch/display/cyede/CMSSW_15_0_10_samplesRun394959_test2.20250730_114159))

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/e81a82a8-e81d-4b67-9a7f-288f1cc3cc5e" />